src/helpers/queries.js

```diff
@@ -0,0 +1,21 @@
+import { Q } from 'cozy-client'
+import { DOCTYPE_CONTACTS } from './doctypes'
+
+export const queryContactById = {
+  definition: id => () => Q(DOCTYPE_CONTACTS).getById(id),
+  options: {
+    as: 'contactById'
```

Comment:
The name should contain the id, otherwise we'll reuse a wrong result if we change the id and add a fetch policy.